.env.example

```
@@ -1,79 +1,126 @@
+#REACT_APP_SWAGGER_SERVER_URL=
+#REACT_APP_GRAPHQL_SERVER_URL=
+#REACT_APP_GRAPHQL_SERVER_PROTOCOL=
+#REACT_APP_GRAPHQL_SERVER_HOST=
+#REACT_APP_GRAPHQL_SERVER_PORT=
+#REACT_APP_CETRIFUGE_PROTOCOL=
+#REACT_APP_CETRIFUGE_HOST=
+#REACT_APP_CETRIFUGE_PORT=
+
+# Metrics and shutdown
+METRICS_ENABLED=false
+GRACEFUL_SHUTDOWN_TIMEOUT=10000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GRACEFUL_SHUTDOWN_TIMEOUT key should go before the METRICS_ENABLED key

(UnorderedKey)